src/main/java/io/gatling/mojo/EnterpriseStartMojo.java

```diff
@@ -78,21 +79,36 @@ public class EnterpriseStartMojo extends AbstractEnterprisePluginMojo {
 
   /**
    * Provides system properties when starting a simulation, in addition to the ones which may
-   * already be configured for that simulation (see
+   * already be defined for that simulation (see
    * https://gatling.io/docs/enterprise/cloud/reference/user/simulations/#step-3-injector-parameters).
+   * To provide system properties in the command line, use the format
+   * -Dgatling.enterprise.simulationSystemProperties=key1=value1,key2=value2
+   */
+  @Parameter private Map<String, String> simulationSystemProperties;
+
+  /**
+   * Alternative to simulationSystemProperties. Use the following format: key1=value1,key2=value2
+   * This is meant to be used in the command line, rather than in the pom.xml
```

Comment:
```suggestion
   * Alternative to simulationSystemProperties. Use the following format: key1=value1,key2=value2.
   * This is meant to be used on the command line, rather than in the pom.xml.
```